simbad/rotsearch/amore_search.py

```diff
@@ -169,11 +169,16 @@ def run(self, models_dir, nproc=2, shres=3.0, pklim=0.5, npic=50, rotastep=1.0,
             name = os.path.basename(dat_model.replace(".dat", ""))
             pdb_struct = simbad.util.pdb_util.PdbStructure()
             pdb_struct.from_file(dat_model)
-            solvent_content = sol_calc.calculate_from_struct(pdb_struct)
-            if solvent_content < min_solvent_content:
-                msg = "Skipping %s: solvent content is predicted to be less than %.2f"
-                logger.debug(msg, name, min_solvent_content)
-                continue
+            try:
```

Comment:
Fair enough for now but would dig into the reason for some of those failing. Worth pinging Andrey an email as well to check with him what's going on.